frontend/src/views/SupplierDashboardView.tsx

```diff
@@ -0,0 +1,255 @@
+/*
+Copyright (c) 2024 Volkswagen AG
+Copyright (c) 2024 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST)
+Copyright (c) 2024 Contributors to the Eclipse Foundation
+
+See the NOTICE file(s) distributed with this work for additional
+information regarding copyright ownership.
+
+This program and the accompanying materials are made available under the
+terms of the Apache License, Version 2.0 which is available at
+https://www.apache.org/licenses/LICENSE-2.0.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+License for the specific language governing permissions and limitations
+under the License.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+import { Input, Table } from '@catena-x/portal-shared-components';
+import { useState } from 'react';
+import { ConfidentialBanner } from '@components/ConfidentialBanner';
+import Autocomplete from '@mui/material/Autocomplete';
+
+type Customer = {
+    name: string;
+    materials: Material[];
+};
+
+type Material = {
+    name: string;
+    demandActual: number[];
+    demandAdditional: number[];
+    production: number[];
+};
+
+const mockCustomerDemands: Customer[] = [
```

Comment:
Supplier Dashboard misses stock information (own and partner), the update functionality of stocks and also limiting information to sites and addresses.

Is this on purpose due to restructuring in future? That were at least the recent changes

Same for outsourcing the information.

Reply:
As aligned: won't follow up